content/en/platform/corda/1.6/cenm/backup-recommendations.md

```diff
@@ -0,0 +1,52 @@
+---
+aliases:
+- /cenm-backup-recommendations.html
+date: '2024-06-12'
+menu:
+  cenm-1-6:
+    identifier: cenm-1-6-cenm-backup-recommendations
+    parent: cenm-1-6-operations
+    weight: 185
+tags:
+- backup
+title: CENM backup recommendations
+---
+
+# CENM backup recommendations
+
+Various components of the Corda platform read their configuration from the file system, and persist data to a database or into files on disk.
+Given that hardware can fail, operators of IT infrastructure must have a sound backup strategy in place. Whilst blockchain platforms can sometimes recover some lost data from their peers, it is rarely the case that a node can recover its full state in this way because real-world blockchain applications invariably contain private information (e.g., customer account information). Moreover, this private information must remain in sync with the ledger state. As such, we strongly recommend implementing a comprehensive backup strategy.
```

Comment:
~~Whilst blockchain platforms can sometimes recover some lost data from their peers, it is rarely the case that a node can recover its full state in this way because real-world blockchain applications invariably contain private information (e.g., customer account information). Moreover, this private information must remain in sync with the ledger state.~~

CENM holds important Network information like approved and revoked network participants as well as the network map and parameters. Whilst the latest network parameters are saved in a config file the rest could not be recovered.